### DISCUSSION TITLE
how to use lightgallery using CDNs?

### DISCUSSION THREAD

**Discussion by Tymur-Popov** [sachinchoolur/lightGallery#1132] [General]
I want to use lightgellery without downloading it in my project, but I can't figure out how

**Comment by sachinchoolur** [comment]
Hey @Tymur-Popov,

What issue are you facing? lightGallery is available on all popular CDNs

Skypack Demo - https://codepen.io/sachinchoolur/pen/qBrbbNQ
Jsdelivr  Demo - https://codepen.io/sachinchoolur/pen/qBrbbNQ
